ios/FluentUI/BottomSheet/BottomSheetViewController.swift

```diff
@@ -242,3 +243,18 @@ public class BottomSheetViewController: UIViewController {
         }
     }
 }
+
+extension BottomSheetViewController: UIGestureRecognizerDelegate {
+    public func gestureRecognizer(_ gestureRecognizer: UIGestureRecognizer, shouldRecognizeSimultaneouslyWith otherGestureRecognizer: UIGestureRecognizer) -> Bool {
+        let shouldRecognizeSimultaneously = false
+        if let contentView = otherGestureRecognizer.view {
+            if let scrollView = contentView as? UIScrollView {
+                // If scroll view has reached the bottom or top bring the bottom sheet pan in action too.
+                if scrollView.contentOffset.y >= (scrollView.contentSize.height - scrollView.frame.size.height) || (scrollView.contentOffset.y <= 0) {
```

Comment:
if so, we should probably check if othergestureregonizer is a pangesture to begin with

Reply:
this makes sense